php/deprecated.sls

```diff
@@ -1,41 +1,31 @@
 # -*- coding: utf-8 -*-
 # vim: ft=sls
 
-{#- Evaluating as `False` by default, using this method since `defaults.yaml` not available in this repo #}
-{%- if not salt['config.get']('php').get('warning_messages', {}).get('v1.0.0', {}).get('mute_critical', False) %}
-php-deprecated-in-v1.0.0-test-succeed:
-  test.succeed_without_changes:
+php-deprecated-in-v1.0.0-test-fail:
+  test.fail_without_changes:
     - name: |
 
-
```

Comment:
Don't remove this line -- we need two blank lines here to match the `deprecated.sls` from `nginx-formula` ([link](https://github.com/saltstack-formulas/nginx-formula/blob/df649c997bc2d4bea7721fbbb54c23efa4688f0e/nginx/deprecated.sls)).  This is best for displaying this message.